rfcs/text/0025-container-metric-fields.md

```diff
@@ -98,6 +339,10 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
+This is a new field set, and the changes introduced will not affect existing ECS implementations.
+
+Integrations or other data sources mapping to ECS will need to map their original events to the new fields as well.
```

Comment:
We can add a point for Kibana metrics UI will be able to display all containers too.

Reply:
Added!!